crates/typst-library/src/diag.rs

```diff
@@ -551,30 +551,49 @@ impl From<PackageError> for EcoString {
 }
 
 /// Format a user-facing error message for an XML-like file format.
-pub fn format_xml_like_error(format: &str, error: roxmltree::Error) -> EcoString {
-    match error {
+pub fn format_xml_like_error(
+    format: &str,
+    error: roxmltree::Error,
+    span: Span,
```

Comment:
same comment on potentially different variable name (not sure if it's 100% applicable to all usages of this function, so it may be disregarded if needed)

Reply:
I renamed all of those spans to `call_span` to make it clearer this is the span of the call site.